scripts/asm-installer/install_asm

```diff
@@ -681,6 +688,16 @@ parse_args() {
         CUSTOM_OVERLAY="${2},${CUSTOM_OVERLAY}"
         shift 2
         ;;
+      --trusted_gcp_project | --trusted-gcp-project)
+        arg_required "${@}"
+        TRUSTED_GCP_PROJECTS="${2},${TRUSTED_GCP_PROJECTS}"
+        shift 2
+        ;;
+      --environ_project_number | --environ-project-number)
+        arg_required "${@}"
+        ENVIRON_PROJECT_NUMBER="$(echo "${2}" | tr '[:upper:]' '[:lower:]')"
+        shift 2
+        ;;
```

Comment:
Is there any reason these can't also be environment variables? I'd rather not add options if we can help it. We can go back and add trusted projects later when it's supported, and ostensibly we should be able to look up the project number via the project ID.

Reply:
yep, that makes sense. will add a patch for it. Also changing this parameters will require more changes in prow tests code. But it is worth doing.

Reply:
ENVIRON_PROJECT_NUMBER is actually required from input to configure multi-project tests in prow. I've moved them into _CI environments to not expose premature configs.